docs/content/en/docs/install/install.md

```diff
@@ -100,24 +94,35 @@ Some helpful hints:
 
 ## Modify Helm configuration options
 
-Helm chart values can be modified before the installation.
-This is useful if you want to install only the `metrics-operator`
-rather than the full Toolkit
-or if you need to change the size of the installation.
-
-To modify configuration options, download a copy of the
-[chart/values.yaml](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn/values.yaml)
-file, modify some values, and use the modified file to install Keptn:
+To modify the Keptn configuration,
+you must modify the appropriate Helm Chart.
+Helm chart values can be modified before the installation
+or can be modified and reapplied after the initial installation.
+This is useful if you do not want to install all components,
+if you need to change the size of the installation,
```

Comment:
what is size of installation ?

Reply:
It was a question that came up once.  I don't want to delay this PR while I chase it so just deleted that line for now.